src/policy.c

```diff
@@ -425,34 +459,39 @@ bool qd_policy_open_lookup_user(
                                                         (PyObject *)policy->py_policy_manager,
                                                         vhost, name_buf, upolicy);
                 if (result2) {
-                    settings->maxFrameSize         = qd_entity_opt_long((qd_entity_t*)upolicy, "maxFrameSize", 0);
-                    settings->maxSessionWindow     = qd_entity_opt_long((qd_entity_t*)upolicy, "maxSessionWindow", 0);
-                    settings->maxSessions          = qd_entity_opt_long((qd_entity_t*)upolicy, "maxSessions", 0);
-                    settings->maxSenders           = qd_entity_opt_long((qd_entity_t*)upolicy, "maxSenders", 0);
-                    settings->maxReceivers         = qd_entity_opt_long((qd_entity_t*)upolicy, "maxReceivers", 0);
-                    if (!settings->allowAnonymousSender) { //don't override if enabled by authz plugin
-                        settings->allowAnonymousSender = qd_entity_opt_bool((qd_entity_t*)upolicy, "allowAnonymousSender", false);
-                    }
-                    if (!settings->allowDynamicSource) { //don't override if enabled by authz plugin
-                        settings->allowDynamicSource   = qd_entity_opt_bool((qd_entity_t*)upolicy, "allowDynamicSource", false);
+                    int truthy = PyObject_IsTrue(result2);
+                    if (truthy) {
+                        settings->maxFrameSize         = qd_entity_opt_long((qd_entity_t*)upolicy, "maxFrameSize", 0);
```

Comment:
Since we are not sending the max-frame-size in the first outgoing open frame, should we populate settings->maxFrameSize which will start enforcing it when the frames start coming in.

Reply:
This function is shared by the incoming connection and outgoing connection handlers. The incoming connection handler needs it.